oauth2/fosite_store_sql.go

```diff
@@ -169,6 +173,26 @@ var migrations = &migrate.MemoryMigrationSource{
 				sqlSchemaDown(sqlTablePKCE, "4"),
 			},
 		},
+		{
+			Id: "5",
+			Up: []string{
+				sqlSchemaUp(sqlTableAccess, "5"),
+				sqlSchemaUp(sqlTableRefresh, "5"),
+			},
+			Down: []string{
+				sqlSchemaDown(sqlTableAccess, "5"),
+				sqlSchemaDown(sqlTableRefresh, "5"),
+			},
+		},
+		{
+			Id: "6",
```

Comment:
These two can be merged, no?

Reply:
@aeneasr I had to keep them separate because I only wanted to create an index on the `requested_at` column on the `hydra_oauth2_access` table. 

I guess I could add a semicolon and add another statement for migration 5?

Reply:
Oh yeah, I see, this is because of the function design. Let's keep it the way you have it